src/SFA.DAS.AdminService.Web/Views/Application/Sequence.cshtml

```diff
@@ -64,11 +64,17 @@
                             <dd class="govuk-summary-list__value">@Model.StandardVersion</dd>
                         </div>
                     }
-                    @if (!string.IsNullOrWhiteSpace(Model.StandardWithReference) && Model.SequenceNo == ApplyConst.STANDARD_WITHDRAWAL_SEQUENCE_NO)
+                    @if (!string.IsNullOrWhiteSpace(Model.StandardName) && Model.SequenceNo == ApplyConst.STANDARD_WITHDRAWAL_SEQUENCE_NO)
```

Comment:
The h1 in the prototype is "Withdrawal application overview" can this be set dynamically with the Sequence number?

Reply:
I think this is driven by what is retrieved from the application in the QnA Api. We've had a few instances of these whereby it's generic because this title also applies for other applications for version, so i think for now we leave it be. 😞